lambda/write-user-services/models.ts

```diff
@@ -0,0 +1,13 @@
+type UrnFdnSub = String;
```

Comment:
Do you think we can move the model objects to a common directory as I have a similar structure when writing the format-record lambda ? Apologies , not 100% sure if that would work or not .

Reply:
Good question! 

Other teams have been talking about this as well - see [this Slack conversation](https://gds.slack.com/archives/C03UDLGC44W/p1665138690298679) for example. The short answer is yes, but we'll need to make the shared objects into an npm module that gets installed by each of the lambdas.

I don't think that should be too difficult though, so we should definitely do it. I'll have a go in a follow up PR